spec/generators/dashboard_generator_spec.rb

```diff
@@ -18,6 +18,15 @@
       expect(dashboard).to have_correct_syntax
     end
 
+    it "should not create file with uninitialized model" do
+      dashboard = file("app/dashboards/customer_dashboard.rb")
+
+      run_generator ["invalid_model"]
+
+      expect(run_generator).to raise_error(NameError)
```

Comment:
The right way to get this to work is to wrap `run_generator` in an `expect { … }`, as the execution of line 24 is what'll throw the exception. Something like this:

```ruby
expect do
  run_generator ["invalid_model"]
end.to raise_error(NameError)
```